easybuild/easyconfigs/f/FastANI/FastANI-1.1-foss-2018b.eb

```diff
@@ -0,0 +1,41 @@
+# easybuild easyconfig
+#
+# John Dey jfdey@fredhutch.org
+
+easyblock = 'ConfigureMake'
+
+name = 'FastANI'
+version = '1.1'
+
+homepage = "http://www.iodbc.org/"
+description = """FastANI is developed for fast alignment-free computation of
+ whole-genome Average Nucleotide Identity (ANI). ANI is defined as mean
+ nucleotide identity of orthologous gene pairs shared between two microbial
+ genomes. FastANI supports pairwise comparison of both complete and draft
+ genome assemblies."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/ParBLiSS/FastANI/archive']
+sources = ['v%(version)s.zip']
+patches = ['FastANI-1.1-memcpy.patch']
```

Comment:
@fizwit Please use `%(version)s` here rather than hardcoding the version number:

```python
patches = ['FastANI-%(version)s-memcpy.patch']